README.md

```diff
@@ -107,3 +128,66 @@ You should be aware that setting this variable provides a safe *subset* of
 functionality; it is still possible that PRs raised for code repositories will
 still fail under the automated CI systems since those systems are running all
 possible tests.
+
+
+## Run the Kata Containers tests
+
+### Requirements to run Kata Containers tests
+
+You need to install the following to run Kata Containers tests:
+
+- [golang](https://golang.org/dl)
+
+  To view the versions of go known to work, see the `golang` entry in the
+  [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml).
+
+- `make`.
+
+### Prepare an environment
+
+The recommended way to set up Kata Containers is using the official and latest
+stable release. You can find the official documentation to do this in the
+[Kata Containers installation user guides](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+If you want to try latest commits of Kata you can use the CI scripts, which will build
+and install from the kata-containers repositories. Follow these steps:
+
+> **Warning:** This may replace/delete packages and configuration that you already have.
+> Please use these steps only on a testing environment.
+
+Add the `$GOPATH/bin` directory to the PATH:
+```
+$ export PATH=${GOPATH}/bin:${PATH}
+```
+
+Clone the `kata-container/tests` repository:
+```
+$ go get github.com/kata-containers/tests
+```
+
+Go to the tests repo directory:
+```
+$ cd $GOPATH/src/github.com/kata-containers/tests
+```
+
+$ Execute the setup script:
+```
+.ci/setup.sh
+```
+
+### Run the tests
+
+If you have already installed the Kata Containers packages and a container
+manager (Docker or Kubernetes) and want to execute the complete tests content,
```

Comment:
Suggested rewrite: 
manager (i.e. Docker or Kubernetes), and you want to execute the content for all the tests, run the following: